reinforcement_learning/rl_cartpole_batch_coach/rl_cartpole_batch_coach.ipynb

```diff
@@ -0,0 +1,726 @@
+{
```

Comment:
<blockquote>As we can see on the above evaluation job, the trained agent run very well by getting total reward around&nbsp;<code>200</code>. We can compare this number with the one we got from the rollouts data, which is around&nbsp;<code>25</code>. In such way we can confirm that the learning agent was getting a better policy from the off-policy data.</blockquote><p>As we can see <strong>from</strong> the above evaluation job, the trained agent <strong>gets a</strong> total reward <strong>of</strong> around&nbsp;<code>200</code> <strong>as compared to total reward of </strong><code><strong>25</strong></code><strong> in our offline dataset</strong>. <strong>Therefore,</strong> we can confirm that the agent <strong>has learned</strong> a better policy from the off-policy data.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/awslabs/amazon-sagemaker-examples/pull/926/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>

Reply:
As we can see from the above evaluation job, the trained agent gets a total reward of around 200 as compared to a total reward around 25 in our offline dataset. Therefore, we can confirm that the agent has learned a better policy from the off-policy data.